src/Operators/Dense.jl

```diff
@@ -195,7 +205,7 @@ function Matrix(Op::GenericEvaluationOperator,n)
     return Op.M(n,n)
 end
 
-function hornermat(x,m)
+function horner_mat(x,m)
```

Comment:
In changing this name, the Fourier code will break.  Let's revert to the old name.